pkg/cmd/server/api/validation/oauth.go

```diff
@@ -102,6 +103,16 @@ func ValidateOAuthConfig(config *api.OAuthConfig, fldPath *field.Path) Validatio
 			)))
 	}
 
+	if config.TokenConfig.AccessTokenTimeoutSeconds != nil {
+		if *config.TokenConfig.AccessTokenTimeoutSeconds < oauthvalidation.ReferenceTimeoutInterval {
```

Comment:
*config.TokenConfig.AccessTokenTimeoutSeconds == 0 is valid

Reply:
fixed